tools/src/main/scala/scala/scalanative/codegen/CodeGen.scala

```diff
@@ -12,6 +12,13 @@ import scalanative.nir._
 
 object CodeGen {
 
+  private final class Platform(target: String) {
+    val isWindows = target.contains("indows")
```

Comment:
You already defined `isWindows`. Note that there is also `Platform.isWindows`. Perhaps, we could merge some of these?